src/util/tutorial.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import type { Disklet } from 'disklet'
+
+const TUTORIAL = 'tutorial.json'
+
+type UserTutorialList = {
+  [userId: string]: {
+    walletListSlideTutorialCount: string
```

Comment:
Because it will be a string anyways when you JSON.stringify it. As it is a JSON so everything will be a string. If passed as a number need to specifically parseInt that number before returning it to the caller.

Reply:
that does not make sense. parseInt  is used on a string. If it is already a number than you never need to call parseInt

sure, Json stringify will turn it into a string but you do not need to deal with that conversion

Reply:
fixed